roles/install_dbserver/defaults/main.yml

```diff
@@ -6,7 +6,6 @@ pg_version: 14
 pg_tuner_version: 1
 pg_owner: "{{ 'enterprisedb' if pg_type == 'EPAS' else 'postgres' }}"
 enable_core_dump: false
```

Comment:
was this intended to be left in place? I noticed they were removed in other roles' defaults

Reply:
> was this intended to be left in place? I noticed they were removed in other roles' defaults

Yeah, I should I should leave `enable_core_dump` in there because it was used in `roles/install_dbserver/tasks/EPAS_RedHat_install.yml`.  Should I handle that differently?

Reply:
no, i think I initially thought this wasn't present, which is why I asked you to include the condition `when: enable_core_dump is defined`, but I see this now. I don't think keeping the conditionals with the `is defined` is an issue

Reply:
Ok, I'll re-edit that `is defined`. ;)

Reply:
you can leave as is if you would like to edit less files :)